src/renderer/containers/Settings/DolphinSettings.tsx

```diff
@@ -68,6 +71,11 @@ export const DolphinSettings: React.FC<{ dolphinType: DolphinLaunchType }> = ({
   return (
     <div>
       <Typography variant="h5">{dolphinTypeName} Dolphin Settings</Typography>
+      <Typography variant="caption">
+        Version: {!dolphinVersion && "Unknown"}
```

Comment:
If you wanted to distinguish between Dolphin not being found and the version being unknown, you should check the status of dolphin and display not found if the status is unknown.
e.g. something like: `const versionString = status === unknown ? "Not found" : !version ? "Unknown" : version;`